purview/api/v1.py

```diff
@@ -23,15 +24,34 @@ def about():
     return jsonify({'version': version})
 
 
+def get_all_relationships(item):
```

Comment:
Can you move this to `utils.py`? Can you add a docstring?

Reply:
Sure